lib/paths.js

```diff
@@ -13,6 +13,7 @@ const {
 } = process.env
 
 export const getPaths = ({ cacheRoot, stateRoot }) => ({
+  secretStore: join(stateRoot, 'secrets'),
```

Comment:
Didn't you say in our call this should be in cacheRoot, because stateRoot might be shared between machines? Or is this path safe after all?

Reply:
```suggestion
  secrets: join(stateRoot, 'secrets'),
```

To me, `paths.secrets` is sufficient. `secretsStore` sounds like an abstraction for storing secrets.

Reply:
> Didn't you say in our call this should be in cacheRoot, because stateRoot might be shared between machines? Or is this path safe after all?

I did say that. Then I looked into `paths.js` and found that we are using `APPLOCALDATA` for storing the state, which means the state stays local - that's exactly what we need.

I think it would be great to capture the requirement for `stateRoot` to be local and not shared with other computers the operator may use, but I don't know how. We can add code comments (where?) and mention this requirement in README too. WDYT?

Reply:
Good idea to add a comment in paths.js and also mention this briefly in the README (I don't think many users will run into this situation where it's shared)

Reply:
Addressed in [c139c6c](https://github.com/filecoin-station/core/pull/424/commits/c139c6c90912d197a7c45909635d02528a3e8411)